recipes/sentry-crashpad/all/conanfile.py

```diff
@@ -68,7 +75,8 @@ def _configure_cmake(self):
         self._cmake.definitions["CRASHPAD_ENABLE_INSTALL"] = True
         self._cmake.definitions["CRASHPAD_ENABLE_INSTALL_DEV"] = True
         self._cmake.definitions["CRASHPAD_ZLIB_SYSTEM"] = True
-        self._cmake.configure()
+        tools.mkdir("cmake-build")
```

Comment:
Please follow our usual convention https://github.com/conan-io/conan-center-index/blob/master/docs/reviewing.md#build-folder

Reply:
Sadly lost of projects break out of source, and it's only a preference not everyone likes to apply it